### ISSUE TITLE
add and subtract times should say to toggle

### ISSUE THREAD

**Issue by podfeet** [podfeet/elapsed-time-adder#17]
the +/- button should say something like "add time, push to switch to subtract time" and vice versa

**Comment by podfeet** [comment]
not super happy with the solution - it says "add button ... press to change to subtract" but doesn't announce the change. if you touch it again it says "subtract button ... press to change to add". tired of fighting with Claude about it. hope it's good enough.
